config/environments/development.rb

```diff
@@ -37,4 +37,27 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.host_name = ENV["DEV_HOST_NAME"] || "localhost:3000"
+
+  # @param options [Hash]
+  # @option options [String] :user
+  # @option options [String] :password
+  # @option options [Symbol] :type
+  #
+  # @return [Boolean, Array] False if a failed authentication, otherwise an
+  #     array of the user's groups
+  config.auth_method = lambda do |options|
```

Comment:
This needs to be added to `test.rb` as well, so it should be defined elsewhere so it can be shared.